doc/Requirements Specification/Parts/Test Cases.lyx

```diff
@@ -99,12 +110,270 @@ Make sure to stay at the level of acceptance testing.
 \end_layout
 
 \begin_layout Plain Layout
-Reference notation is /T#/
+Reference notation is /T#/.
+ /OT#/ indicates test for optional requirements.
+\end_layout
+
+\end_inset
+
+
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/T10/"
+
+\end_inset
+
+ Smoke testing.
+ First of all, the software has to be running, otherwise there is nothing
+ to test.
+ On account of this the first test is a simple run-through.
+ For a valid input this has to work without exceptions and the software
+ has to terminate.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/T20/"
+
+\end_inset
+
+ Correct measurement (correct results)
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/T30/"
+
+\end_inset
+
+ Assert that it works for a system with only the software specified in 
```

Comment:
What "it" is might be obvious for us but not necessarily for the reader of the document.